index.html

```diff
@@ -0,0 +1,300 @@
+        </ul>
+    </div>
+      </section>
+
+      <section  id="card" class="cards">
+        <div class="addmore"></div>
+        <!-- <div class="container">
+            <img src="/img/project-img/d-project-1.png" alt="">
+        <div>
+        <h2 class="card-heading">Tonic</h2>
+        <ul class="prof">
+            <li class="item">CANOPY </li>
+            <li class="item-prof">&#9679; Back End Dev  &#9679;  </li>
+            <li class="item-prof">2015</li>
+        </ul>
+        <p class="dec-p">A daily selection of 
+            privately personalized reads; no accounts
+             or sign-ups required.</p>
+            <ul class="lang-tag">
+                <li class="lang-list">HTML</li>
+                <li class="lang-list">CSS</li>
+                <li class="lang-list">JavaScript</li>
+            </ul>
+        <button  id="click-btn"class="see-btn" >see project</button>
+    </div>  
+</div>
+
+<div class="container" id="one-desk">
+    <img src="/img/project-img/d-project-2.png" alt="">
+<div 
+<h2 class="card-heading">Multi-Post
+Stories</h2>
+<ul class="prof">
+    <li class="item">CANOPY </li>
+    <li class="item-prof">&#9679; Back End Dev  &#9679;  </li>
+    <li class="item-prof">2015</li>
+</ul>
+<p class="dec-p">A daily selection of 
+    privately personalized reads; no accounts
+     or sign-ups required.</p>
+    <ul class="lang-tag">
+        <li class="lang-list">HTML</li>
+        <li class="lang-list">CSS</li>
+        <li class="lang-list">JavaScript</li>
+    </ul>
+<button class="see-btn">see project</button>
+</div>  
+</div>
+
+<div class="container">
+    <img src="/img/project-img/d-project-3.png" alt="">
+<div>
+<h2 class="desk-card">Facebook 360</h2>
+<h2 class="card-heading" >Tonic</h2>
+<ul class="prof">
+    <li class="item">CANOPY </li>
+    <li class="item-prof">&#9679; Back End Dev  &#9679;  </li>
+    <li class="item-prof">2015</li>
+</ul>
+<p class="dec-p">A daily selection of 
+    privately personalized reads; no accounts
+     or sign-ups required.</p>
+    <ul class="lang-tag">
+        <li class="lang-list">HTML</li>
+        <li class="lang-list">CSS</li>
+        <li class="lang-list">JavaScript</li>
+    </ul>
+<button class="see-btn">see project</button>
+</div>  
+</div>
+
+<div id="mylast-card" class="container" >
+    <img src="/img/project-img/d-project-4.png" alt="">
+<div>
+<div class="dent">
+<h2 class="card-heading">Multi-Post
+  Stories</h2>
+<ul class="prof">
+    <li class="item">CANOPY </li>
+    <li class="item-prof">&#9679; Back End Dev  &#9679;  </li>
+    <li class="item-prof">2015</li>
+</ul>
+<p class="dec-p">A daily selection of 
+    privately personalized reads; no accounts
+     or sign-ups required.</p>
+    <ul class="lang-tag">
+        <li class="lang-list">HTML</li>
+        <li class="lang-list">CSS</li>
+        <li class="lang-list">JavaScript</li>
+    </ul>
+<button class="see-btn">see project</button>
+</div>  
+<!-- </div> -->
```

Comment:
- [ ] Kindly ensure that you remove all commented-out/ unused/ unwanted pieces of code from your code base. It makes your codebase look messy and cluttered. Having commented out code also violates the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).